JSQMessagesViewController/Views/JSQMessagesTypingIndicatorFooterView.m

```diff
@@ -21,17 +21,31 @@
 #import "JSQMessagesBubbleImageFactory.h"
 
 #import "UIImage+JSQMessages.h"
+#import "UIColor+JSQMessages.h"
 
 const CGFloat kJSQMessagesTypingIndicatorFooterViewHeight = 46.0f;
 
+@interface JSQMessagesTypingIndicatorAnimator : NSObject
+- (id) initWithTypingIndicatorView:(JSQMessagesTypingIndicatorFooterView*) view :(UIColor*) ellipsisColor;
+@property (nonatomic, strong) JSQMessagesTypingIndicatorFooterView *typingView;
+@property (nonatomic, strong) UIColor *ellipsisColor;
+@property (nonatomic, assign) BOOL animating;
+- (void) startAnimating;
+- (void) stopAnimating;
+@end
```

Comment:
once in a new file, let's clean up this formatting to be consistent with the rest of the project :smile: